--- adms-2.3.2/configure.ac.orig	2014-07-20 12:14:38.813866179 +0100
+++ adms-2.3.2/configure.ac	2014-07-20 12:15:47.073986623 +0100
@@ -24,7 +24,7 @@
 # Automake Initialisation.
 AM_INIT_AUTOMAKE
 
-# Maintainer mode is used to detect/run Perl (XML::LibXml and GD modules) and make dist tarball
+# Maintainer mode is used to detect/run Perl (XML::LibXml module) and make dist tarball
 AM_MAINTAINER_MODE
 
 AX_PROG_FLEX([],[AC_MSG_WARN(Program flex not found (found $LEX))
@@ -84,7 +84,7 @@
 AC_PROG_LIBTOOL
 AC_SUBST([LIBTOOL_DEPS])
 
-dnl Perl, XML::LibXML and GD modules required for maintainer
+dnl Perl, XML::LibXML module required for maintainer
 if test $USE_MAINTAINER_MODE = yes; then
 
   AC_PATH_PROG(PERL, perl, :)
@@ -95,8 +95,6 @@
   AX_PERL_MODULE_VERSION([XML::LibXML 2],[],[
       AC_MSG_RESULT(failed)
       AC_MSG_ERROR([Perl package XML::LibXML may be downloaded from http://search.cpan.org/dist/libXML])])
-  AX_PERL_MODULE_VERSION([GD 2.4],[],[
-      AC_MSG_ERROR([Perl package GD:: may be downloaded from http://search.cpan.org/dist/GD])])
 fi
 
 AC_CONFIG_FILES([Makefile
--- adms-2.3.2/images/Makefile.am.orig	2014-07-20 12:14:23.428613664 +0100
+++ adms-2.3.2/images/Makefile.am	2014-07-20 12:16:24.456600306 +0100
@@ -8,10 +8,4 @@
 
 EXTRA_DIST=mkimages.pl $(generated_FILES)
 
-# Perl GD module only required for maintainers
-if MAINTAINER_MODE
-$(generated_FILES): ${srcdir}/mkimages.pl
-	$(PERL) ${srcdir}/mkimages.pl
-endif
-
 CLEANFILES=core .*.swp *\~ $(generated_FILES)
