--- adms-2.3.2/admsXml/preprocessorLex.l.orig	2014-07-20 15:39:43.139480510 +0100
+++ adms-2.3.2/admsXml/preprocessorLex.l	2014-07-20 15:40:34.816289041 +0100
@@ -85,7 +85,7 @@
 static void adms_file_fprintf ( char *filename, const char *filedata)
 {
   FILE *ofh = fopen (filename,"w");
-  if (fprintf (ofh, filedata)==EOF)
+  if (fprintf ("%s", ofh, filedata)==EOF)
     adms_message_fatal(("%s: cannot write file\n", filename))
   fclose (ofh);
 }
